### PR TITLE
Feat: Add no public transit routes found messages

### LINKED ISSUE #599: Change message if no public tranfer routes are found
Originally reported here https://github.com/stadtnavi/digitransit-ui/issues/542 and https://github.com/stadtnavi/digitransit-ui/issues/503

Case 1) The following message should be changed to "No route using public transport only found."
grafik

Case 2) as reported in the user feedback - the following message should also be changed if a car or bike route were found and the system shows public transport alternatives.
Existing message: "No route suggestions were found with the your settings. However, we found the following route options:"
New message: "Only routes without public transport were found with your settings. However, we suggest you consider the following route options:"
To reproduce this case please disable all modes from settings except Car mode
https://herrenberg.beta.stadtnavi.eu/reiseplan/Vorsteigstra%C3%9Fe%2C%2070193%20Stuttgart%3A%3A48.7812373%2C9.1553375/Kocherstra%C3%9Fe%2C%2071083%20Oberjesingen%3A%3A48.6254176%2C8.8381856?time=1622828190

### PR DESCRIPTION
**Pull request by andreashelms**
## Proposed Changes

  - Add no public transit routes found messages

## Pull Request Check List

  - A reasonable set of unit tests is included
  - Console does not show new warnings/errors
  - Changes are documented or they are self explanatory
  - This pull request does not have any merge conflicts
  - All existing tests pass in CI build
  - Code coverage does not decrease (unless measured incorrectly)

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform
  - Check that the implementation matches the design, when such one is defined in a Jira issue
  - Merge the pull request

closes #599

### CODE REVIEW

File: app/translations.js [hunk 1]
```diff
@@ -955,6 +955,7 @@ const translations = {
     'walk-bike-itinerary-1': 'Ihre Suche ergab nur Fußweg-Routen.',
     'walk-bike-itinerary-2': 'Ihre Suche ergab nur Fahrrad-Routen.',
     'walk-bike-itinerary-3': 'Ihre Suche ergab nur Fußweg- und Fahrrad-Routen.',
+    'car-park-ride-itinerary': 'Es wurde keine Route gefunden, die ausschließlich öffentliche Verkehrsmittel nutzt.',
```
Comment (andreashelms) [inline app/translations.js:959]: Please check translation

File: app/translations.js [hunk 2]
```diff
@@ -675,7 +675,7 @@ const translations = {
     'no-route-origin-same-as-destination':
       'Keine Reisevorschläge gefunden, da Abfahrts- und Zielort übereinstimmen.',
     'no-route-showing-alternative-options':
-      'Keine Routenvorschläge mit Ihren Einstelllungen gefunden. Stattdessen haben wird die folgenden Reiseoptionen gefunden:',
+      'Es wurden keine Routen mit öffentlichen Verkehrsmitteln mit Ihren Einstelllungen gefunden. Prüfen Sie, ob die folgenden Reiseoptionen für Sie passend sind:',
```
Comment (andreashelms) [inline app/translations.js]: Please check translation

**Comment by leonardehrenfried** [comment]
This is actually a better solution that I envisioned - well done!

**Comment by leonardehrenfried** [comment]
There is one strange edge that we still need to handle.

https://herrenberg.stadtnavi.de/reiseplan/Schillerstra%C3%9Fe%206%2C%2071083%20Herrenberg%3A%3A48.592050872804194%2C8.865108489990236/Marktplatz%2C%2071083%20Herrenberg%3A%3A48.59639357467958%2C8.870151042938234?time=1625088600

Screenshot from 2021-06-30 10-48-32

You can have walk, bike (or any combination of those) together with Ruftaxi.

In this case I would suggest the following message: 

DE: "Nur Routen mit Ruftaxi-Anteil gefunden."
EN: "Only routes involving on-demand taxis found."

This isn't 100% true (as there might be cycling and walking routes) but let's not overcomplicate it.

**Pull request merged**